frameworks/react/__tests__/AdvancedPicture.test.tsx

```diff
@@ -25,9 +27,10 @@ describe('AdvancedPicture', () => {
       expect(component.html()).toContain(
         '<picture>' +
         '<img src="https://res.cloudinary.com/demo/image/upload/sample">' +
-        '<source srcset="https://res.cloudinary.com/demo/image/upload/c_crop,w_500/dog" sizes="80vw"' +
-        ' media="(min-width: 500px) and (max-width: 800px)">' +
-        '</picture>');
+        '<source media="(min-width: 500px) and (max-width: 800px)" sizes="80vw" ' +
+        'srcset="https://res.cloudinary.com/demo/image/upload/c_crop,w_500/dog">' +
```

Comment:
`c_crop,w_500` is very confusing in these tests. I know it comes with the default image. It's just harder to follow.

Reply:
updated and removed transformation